packages/expo-crypto/src/Crypto.ts

```diff
@@ -142,3 +142,22 @@ export async function digestStringAsync(
 
   return await ExpoCrypto.digestStringAsync(algorithm, data, options);
 }
+
+/**
+ * The `getRandomValues()` method of `Crypto` fills a provided `TypedArray` with cryptographically secure random values.
+ *
+ * @param typedArray A `TypedArray` to fill with cryptographically secure random values.
+ * @return The same array that was passed in, but filled with cryptographically secure random values.
+ *
+ * @example
+ * ```ts
+ * Crypto.getRandomValues(new Uint8Array(16));
+ * ```
+ */
+export function getRandomValues(typedArray: TypedArray): TypedArray {
+  if (!ExpoCrypto.getRandomValues) {
+    throw new UnavailabilityError('expo-crypto', 'getRandomValues');
+  }
```

Comment:
We don't need this anymore in synchronous functions. `ExpoCrypto.getRandomValues` will throw automatically with the similar error. In asynchronous functions we could drop that too, but theoretically we still support remote debugger in which case our modules are in bridge mode and so it's not possible for us to throw something from the native side if the function is not defined.

```suggestion
```